matelint/src/rules/indents-for-attrs.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
Use ESM import/export system.

As we want to extend linter, there should be type how to define a rule and linter should accept only correct rules